docs/tutorials/Martini3/Small_Molecule_Parametrization_FastForward/index.qmd

```diff
@@ -0,0 +1,264 @@
+
+``` bash
+cd ../4_CG_sim
+```
+``` bash
+cp ../3_initial_CG/ENAP.itp .
+```
+``` bash
+bash run_CG.sh
+```
+
+where the command will run an energy-minimization, followed by an NPT equilibration, and by an MD run of 100 ns (inspect the script and the various `.mdp` files to know more) for the Martini system in water. Finally, a ligand-only trajectory and topology are generated for comparison to the mapped ones made earlier.
+
+### 6. Optimize CG bonded parameters
+<hr>
+
+You can now compare the simulated distributions to the reference ones using the final program of the fast_forward package, `ff_assess`. Use the following command:
+
+``` bash
+ff_assess -f prod/lig_CG.xtc -s prod/lig_CG.tpr -i ENAP.itp -d ../3_initial_CG/ -plots
+```
+
+The `ff_assess` program simply uses the annotated topology of the molecule to calculate the interactions actually obtained during the CG simulation, and plots them against the reference distributions previously generated by `ff_inter` (given in a file path to the `-d` flag). Finally, a score is generated using the [Hellinger distance](https://en.wikipedia.org/wiki/Hellinger_distance) comparing the simulated and mapped distributions.
+
+The plot produced should look like this:
+
+<!-- insert figure with html to modify width -->
+<figure>
+    <center>
+    <img src="fig3.png" alt="Figure 3 | CG simulation vs. mapped bonded distributions." style="width:100%">
+    <figcaption>Figure 3 | CG simulation vs. mapped bonded distributions.</figcaption>
+    </center>
+</figure>
+The agreement is very good, noted from both the figure above and also the report of the distribution distance scores generated.
+
+Note that the bimodality of the distributions of the first two dihedrals cannot be captured by the CG model. However, the size of the CG distribution will seemingly capture the two AA configurations into the single CG configuration. If the agreement is not satisfactory at the first iteration - which is likely to happen - you should play with the equilibrium value and force constants in the CG `.itp` and iterate till satisfactory agreement is achieved.
+
+### 7. Comparison to experimental results, further refinements, and final considerations
+<hr>
+
+As [before](../Small_Molecule_Parametrization/index.qmd#comparison-to-experimental-results-further-refinements-and-final-considerations) it is imperative that any newly parameterised molecule for the Martini force field is validated against as many references as possible. The log P and SASA are
+
+
+#### 7.1 Molecular volume and shape
+<hr>
+
+To automate the creation of a vdwradii.dat for a CG molecule, a small Python script, `make_cg_sasa_file.py` is provided with this tutorial to automate the process. Otherwise, the steps and scripts from the previous tutorial can be followed again.
```

Comment:
```suggestion
To automate the creation of a vdwradii.dat for a CG molecule, a small Python script, `make_cg_sasa_file.py` is provided with this tutorial to automate the process. Otherwise, the steps and scripts [from the previous tutorial](../Small_Molecule_Parametrization/index.qmd#molecular-volume-and-shape) can be followed again.
```